pavelib/assets.py

```diff
@@ -46,17 +47,25 @@
 # A list of NPM installed libraries that should be copied into the common
 # static directory.
 NPM_INSTALLED_LIBRARIES = [
-    'jquery/dist/jquery.js',
-    'jquery-migrate/dist/jquery-migrate.js',
-    'jquery.scrollto/jquery.scrollTo.js',
-    'underscore/underscore.js',
-    'underscore.string/dist/underscore.string.js',
-    'picturefill/dist/picturefill.js',
+    'backbone-validation/dist/backbone-validation-min.js',
     'backbone/backbone.js',
     'edx-ui-toolkit/node_modules/backbone.paginator/lib/backbone.paginator.js',
-    'backbone-validation/dist/backbone-validation-min.js',
     'edx-ui-toolkit/node_modules/moment-timezone/builds/moment-timezone-with-data.js',
     'edx-ui-toolkit/node_modules/moment/min/moment-with-locales.js',
+    'jquery-migrate/dist/jquery-migrate.js',
```

Comment:
This is the list of files that are installed via NPM, and that we need to copy into the static vendor directory to have them consumed by the pipeline.  That's why both moment-with-locales and backbone.paginator are in this list, as well as RequireJS which I'm now getting from NPM too.

I separated out the developer libraries below because those files need to be available when running tests, but they are not installed on production machines.

Note: this is an ugly solution that we implemented to replace the previous ugly solution of using symlinks. I have a JIRA ticket to come up with a better solution:

https://openedx.atlassian.net/browse/TNL-6001